prac_09/silver_service_taxi.py

```diff
@@ -0,0 +1,17 @@
+from prac_09.taxi import Taxi
```

Comment:
You can just directly call taxi file since the program file is in the same file as the taxi file